content/news/release-20180217.md

```diff
@@ -0,0 +1,59 @@
+ * The Global Chat has been removed from the server browser and lobby
+ * A chat dialog has been added to the in-game menu
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI.
+</div>
+
+Of course, we realize that the vast majority of OpenRA players do not play online. We haven't forgotten about you either!
+Improvements for the singleplayer experience include:
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it uses superweapons and support powers
+ * Several other changes that should improve the AI behaviour and performance
+
+<div style="text-align:center" markdown="1">
+![AI Airstrike](/images/news/20180102-airstrike.png)
+
+Improvements to the AI support power targeting mean that this will be a rarer sight.
+ </div>
+
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target enemies revealed by the Red Alert GPS Satellite
+* A collection of community-driven balance changes for Red Alert
+* Fixes for many graphical glitches and polish issues
+* Fixes for several platform integration issues on Windows and macOS
+* Further improvements to memory usage and performance
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-mapoptions.png" width="600" alt="Expanded game options">
+
+The Map Options tab includes several new settings, and can be customized by scripted maps.
+</div>
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-winmacos.png" width="600" alt="Windows and macOS improvements">
+
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+As always, we suggest you read the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/583535f22b30d8f744d55fe98f23345fed4d3664) if you'd like more information on the full list of changes.
+We make these playtests available to get feedback from the community about the future direction of OpenRA, and we value your feedback.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
+
+This development cycle brought significant improvements for community mods, but we recommend that modders wait for the next release before updating their projects.  We plan to merge a new version of our semi-automated rule upgrader before the next release, which will simplify the process of updating your [Mod SDK](https://github.com/OpenRA/OpenRAModSDK) based projects to the latest engine version. Stay tuned for more info!
```

Comment:
Better to remove this.

Reply:
Just the "stay tuned for more info" or the whole upgrader paragraph?

Reply:
The whole paragraph.  I have been disheartened by the poor community response to the SDK and to us in general, and do not plan to invest any more of my increasingly scarce time in supporting them.  It would be misleading to string people along unless/until somebody is willing to maintain it.